package-system/libpng/Findlibpng.cmake

```
@@ -35,8 +35,15 @@ set(libpng_LIBRARY                   "$<$<CONFIG:profile>:${libpng_LIBRARY_RELEA
 set(libpng_LIBRARY ${libpng_LIBRARY} "$<$<CONFIG:Release>:${libpng_LIBRARY_RELEASE}>")
 set(libpng_LIBRARY ${libpng_LIBRARY} "$<$<CONFIG:Debug>:${libpng_LIBRARY_DEBUG}>")
 
-add_library(${TARGET_WITH_NAMESPACE} INTERFACE IMPORTED GLOBAL)
+add_library(${TARGET_WITH_NAMESPACE} STATIC IMPORTED GLOBAL)
 ly_target_include_system_directories(TARGET ${TARGET_WITH_NAMESPACE} INTERFACE ${${MY_NAME}_INCLUDE_DIR})
-target_link_libraries(${TARGET_WITH_NAMESPACE} INTERFACE ${libpng_LIBRARY})
+# "Because libpng depends on the zlib, zlib needs to be added as a link dependency of the libpng target, which must be a STATIC IMPORTED target with an IMPORTED_LOCATION_$<CONFIG> set to be library files."
+set_target_properties(${TARGET_WITH_NAMESPACE}
+    PROPERTIES
+        IMPORTED_LOCATION_DEBUG ${${MY_NAME}_LIBRARY_DEBUG}
+        IMPORTED_LOCATION_PROFILE ${${MY_NAME}_LIBRARY_RELEASE}
+        IMPORTED_LOCATION_RELEASE ${${MY_NAME}_LIBRARY_RELEASE}
```

After discussion we could collapse the `IMPORTED_LOCATION_PROFILE` and `IMPORTED_LOCATION_RELEASE` to a single `IMPORTED_LOCATION` property that references the Release library

```suggestion
        IMPORTED_LOCATION ${${MY_NAME}_LIBRARY_RELEASE}
```